README.md

```diff
@@ -243,27 +215,29 @@ Gateway settings:
 
   1. **ValueId Topics**: _Automatically configured_. The topic where zwave values are published will be:
 
-     `<mqtt_prefix>/<?node_location>/<node_id>/<class_id>/<instance>/<index>`
+     `<mqtt_prefix>/<?node_location>/<nodeId>/<commandClass>/<endpoint>/<property>/<propertyKey>`
```

Comment:
Are these the same just different terms? Probably worth explaining that to help with the transition

Reply:
There have been a long conversation on slack about this, them are not really the same, I will add a link to zwave-js docs once the "Migrating from OZW" section will speak about this more in deep :)